src/util/mod.rs

```diff
@@ -387,7 +387,7 @@ impl<L, R> Either<L, R> {
 }
 
 /// `build_info` returns a tuple of Strings that contains build utc time and commit hash.
-fn build_info() -> (String, String) {
+pub fn build_info() -> (String, String) {
```

Comment:
the -v/--version flag is just for the convenience of cli or bash script. And we expect the -v flag be used alone (like the -h flag). So we don't need to bother the log system but do some println!.